### ISSUE TITLE
Brakeman fails to load on Ruby 3.0

### ISSUE THREAD

**Issue by ryanwi** [presidentbeef/brakeman#1539]
### Background

Brakeman version: 4.10.0
Rails version: 6.1.0
Ruby version: 3.0.0

Link to Rails application code: https://github.com/ryanwi/rails-on-docker

### Issue

What problem are you seeing? 

Brakeman fails to load on Ruby 3.0.

```
bundler: failed to load command: brakeman (/home/runner/work/rails-on-docker/rails-on-docker/vendor/bundle/ruby/3.0.0/bin/brakeman)
/home/runner/work/rails-on-docker/rails-on-docker/vendor/bundle/ruby/3.0.0/gems/brakeman-4.10.0/lib/ruby_parser/bm_sexp_processor.rb:82:in `process': Result must be a Sexp, was Array:[:arglist, s(:str, "1.0")] (SexpTypeError)
```

**Comment by jeremy** [comment]
Fixed by #1538

**Comment by ryanwi** [comment]
Great! Thank you

**Comment by presidentbeef** [comment]
This is fixed in [4.10.1](https://github.com/presidentbeef/brakeman/releases/tag/v4.10.1). 🎅🎄
